test/parallel/test-buffer-indexof.js

```diff
@@ -31,10 +31,10 @@ assert.strictEqual(b.indexOf('bc', -Infinity), 1);
 assert.strictEqual(b.indexOf('bc', Infinity), -1);
 assert.strictEqual(b.indexOf('f'), b.length - 1);
 assert.strictEqual(b.indexOf('z'), -1);
-assert.strictEqual(b.indexOf(''), -1);
-assert.strictEqual(b.indexOf('', 1), -1);
-assert.strictEqual(b.indexOf('', b.length + 1), -1);
-assert.strictEqual(b.indexOf('', Infinity), -1);
+assert.strictEqual(b.indexOf(''), 0);
+assert.strictEqual(b.indexOf('', 1), 0);
```

Comment:
To match the behavior of `String.prototype.indexOf()`, this should return `1` rather than `0`.